### DISCUSSION TITLE
Saving Progress

### DISCUSSION THREAD

**Discussion by wehrenberg1** [HEXRD/hexrdgui#1385] [Q&A]
What's the best way to save my analysis so that I can easily bring it back up at a later time?  Do I need to save all 4 of the images, state, configuration, and materials?

**Comment by psavery** [comment]
@wehrenberg1 The state file actually contains in it not only the GUI configuration, but also the images, instrument configuration, and materials.

So the state file is the way you can save your whole analysis. Note that if you are starting the program from the command line, you can also load a state file on startup via `hexrdgui ~/path/to/state.h5`
